src/IssueViz.Security/Hotspots/HotspotsList/HotspotsControl.xaml

```diff
@@ -2,8 +2,6 @@
              x:ClassModifier="internal"
              xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
              xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-             xmlns:core="clr-namespace:SonarLint.VisualStudio.Core.WPF;assembly=SonarLint.VisualStudio.Core"
```

Comment:
Moved some more resources to the shared resources. I manually tested that hotspots navigability still works as expected.